packages/assistify-ai/server/proxySmarti.js

```diff
@@ -0,0 +1,72 @@
+/* globals RocketChat */
+
+const verbs = {
+	get: 'GET',
+	post: 'POST',
+	put: 'PUT',
+	delete: 'DELETE'
+};
+
+function propagateToSmarti(method, path, body) {
+
+	const SMARTI_URL = RocketChat.settings.get('DBS_AI_Smarti_URL').replace(/\/?$/, '/');
+	const URL = `${ SMARTI_URL }${ path }`;
+
+	//get the target Smarti-URL, add the client secret to the header and send the request to Smarti
+	try {
+		this.logger.debug(method, 'to', URL);
```

Comment:
`this.logger` is undefined and therefore throwing an exception.
How is the recommended way in RC to log?